OpenEdXMobile/src/main/java/org/edx/mobile/services/CourseManager.java

```diff
@@ -134,4 +134,19 @@ public boolean apply(CourseComponent courseComponent) {
             }
         });
     }
+
+    @Nullable
+    public CourseComponent getCourseComponentById(@NonNull String blocksApiVersion,
+                                                  @NonNull String courseId,
+                                                  @NonNull String componentId) {
+        CourseComponent courseComponent = getCourseDataFromPersistableCache(blocksApiVersion, courseId);
+        if (courseComponent == null)
+            return null;
+        return courseComponent.find(new Filter<CourseComponent>() {
+            @Override
+            public boolean apply(CourseComponent courseComponent) {
+                return componentId.equals(courseComponent.getId());
+            }
+        });
+    }
```

Comment:
this method is unused so we can remove this.

Reply:
removed